zip-0316.rst

```diff
@@ -521,14 +560,14 @@ zero bytes, to the raw encoding, then applies :math:`\mathsf{F4Jumble}`
 before encoding the result with Bech32m.
 
 The Consumer rejects any Bech32m-decoded byte sequence that is less than
-48 bytes or greater than 16448 bytes; otherwise it applies
-:math:`\mathsf{F4Jumble}^{-1}`. It rejects any result that does not end
+48 bytes or greater than 4194368 bytes; otherwise it applies
```

Comment:
Where does this constant arise from? Could this constant be named and associated with the function that derives the bound?

Reply:
TODO (@daira )